<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QDialog" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1922</width>
    <height>990</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/back36.jpg);</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>240</y>
     <width>170</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/xen.jpeg);</string>
   </property>
   <property name="title">
    <string>Textile</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>130</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string>Okay</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>240</y>
     <width>170</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/xen.jpeg);</string>
   </property>
   <property name="title">
    <string>Civil</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>130</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string>Okay</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>560</y>
     <width>170</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/xen.jpeg);</string>
   </property>
   <property name="title">
    <string>Physcis</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>130</width>
      <height>170</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:&quot;C:/Users/LENOVO/Documents/xenesis_1/cs.jpg&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>560</y>
     <width>170</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/xen.jpeg);</string>
   </property>
   <property name="title">
    <string>Maths</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>130</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>560</y>
     <width>170</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/xen.jpeg);</string>
   </property>
   <property name="title">
    <string>Chemistry</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>130</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_10">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>560</y>
     <width>170</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/xen.jpeg);
</string>
   </property>
   <property name="title">
    <string>Managment</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>130</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>170</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/xen.jpeg);</string>
   </property>
   <property name="title">
    <string>CS and IT</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>20</y>
      <width>131</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string>Okay</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>240</y>
     <width>170</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/xen.jpeg);</string>
   </property>
   <property name="title">
    <string>Mechenical</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>130</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Okay</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>560</y>
     <width>170</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/xen.jpeg);</string>
   </property>
   <property name="title">
    <string>Novel</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>130</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>240</y>
     <width>170</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/xen.jpeg);</string>
   </property>
   <property name="title">
    <string>Electrical</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>130</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string>Okay</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>1151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-image: url(:/images/back45.jpg);</string>
   </property>
   <property name="text">
    <string>  Welcome To Xenesis</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>-10</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
padding :0 8px;
background: light red;
border-radius:10px;
}</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>1580</x>
     <y>-10</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

padding :0 8px;
background: light red;
border-radius:10px;
}</string>
   </property>
   <property name="text">
    <string>About US</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>1740</x>
     <y>-10</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

padding :0 8px;
background: light red;
border-radius:10px;
}</string>
   </property>
   <property name="text">
    <string>Webservices</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>440</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover:!pressed

{
background:grey;
}
QToolButton
{
border-radius:10px;
background: rgb(0, 170, 0);


}
</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>440</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover:!pressed

{
background:grey;
}
QToolButton
{
border-radius:10px;
background: rgb(0, 170, 0);


}
</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>440</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover:!pressed

{
background:grey;
}
QToolButton
{
border-radius:10px;
background: rgb(0, 170, 0);


}
</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>440</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover:!pressed

{
background:grey;
}
QToolButton
{
border-radius:10px;
background: rgb(0, 170, 0);


}
</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>440</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover:!pressed

{
background:grey;
}
QToolButton
{
border-radius:10px;
background: rgb(0, 170, 0);


}</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>770</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover:!pressed

{
background:grey;
}
QToolButton
{
border-radius:10px;
background: rgb(0, 170, 0);


}

</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>780</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover:!pressed

{
background:grey;
}
QToolButton
{
border-radius:10px;
background: rgb(0, 170, 0);


}</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>780</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover:!pressed

{
background:grey;
}
QToolButton
{
border-radius:10px;
background: rgb(0, 170, 0);


}</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>780</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover:!pressed

{
background:grey;
}
QToolButton
{
border-radius:10px;
background: rgb(0, 170, 0);


}
</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>780</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover:!pressed

{
background:grey;
}
QToolButton
{
border-radius:10px;
background: rgb(0, 170, 0);


}</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
